backend/model/model.go

```diff
@@ -641,7 +641,8 @@ type Session struct {
 	VerboseID             string  `json:"verbose_id"`
 
 	// Excluded will be true when we would typically have deleted the session
-	Excluded bool `gorm:"default:false"`
+	Excluded       bool `gorm:"default:false"`
+	ExcludedReason modelInputs.SessionExcludedReason
```

Comment:
if the session is not exclued, what is the value? should this be a pointer and be set to nil?